src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/TagHelperSemanticRangeVisitor.cs

```diff
@@ -573,8 +574,8 @@ private void AddSemanticRange(SyntaxNode node, int semanticKind)
 
         void AddRange(SemanticRange semanticRange)
         {
-            if (semanticRange.StartLine == semanticRange.EndLine &&
-                semanticRange.StartCharacter == semanticRange.EndCharacter)
+            var linePositionSpan = semanticRange.AsLinePositionSpan();
+            if (linePositionSpan.Start != linePositionSpan.End)
```

Comment:
Isn't this backwards? It used to exit early if they were equal.

Reply:
Thanks for making debugging the test failures easy 😄.